themes/vuetober/src/components/Teacher.vue

```diff
@@ -72,9 +72,22 @@
                       :key="feedback.id"
                       class="d-flex align-items-center mb-4 border-0 shadow"
                     >
-                      <div class="p-2">
+                      <div class="p-2 mr-auto">
                         {{ feedback.feedback }}
                       </div>
+                      <p class="font-weight-bold my-0 pr-2">
+                        {{ feedback.likes.length }}
+                      </p>
+                      <b-link @click="like(feedback)">
+                        <b-icon-heart
+                          variant="danger"
+                          v-if="!feedback.likes.some((f) => +f.user_id === +user.id)"
```

Comment:
Preco `+`?

Reply:
pretoze octotber mi vracia id ako string, potrebujem ho porovnavat ako int potom

Reply:
id by mali by vzdy stringy. Mozes spravit aby sa  `user.id` ukladal vzdy ako string? To by bolo idealne riesenie.

A ked uz potrebujes convertovat string na number, odporucam bud `Number` alebo `parseInt`. 

`+` na konvertovanie na `number` je dost obskurna funkcionalita a nie je jasne na prvy pohlad co to robi, bez toho aby niekto googlil dokumentaciu. Napriklad ani ja som si nepamatal ci to konvertuje `string` na `number` alebo naopak. Z `Number` alebo `parseInt` je to jasne hned, co sa deje.

Reply:
Okej, dik. Ja som bol cely cas v tom ze idcka maju byt int.